pandas/tests/groupby/test_categorical.py

```diff
@@ -558,9 +558,13 @@ def test_as_index():
     result = df.groupby(['cat', s], as_index=False, observed=True).sum()
     tm.assert_frame_equal(result, expected)
 
-    # GH18872: conflicting names in desired index
-    with pytest.raises(ValueError):
+    # GH 19029: conflicitng names should not raise a value error anymore
+    raised = False
```

Comment:
Rather than doing this you should say what the `expected` object is and compare to that the result of the groupby